FAQ.md

```diff
@@ -10,26 +10,43 @@ Feel free to add new questions or send comments to this page as pull requests._
+
+I don't expect static IPs would be needed as I don't expect to be hosting services to the outside world. Of course, if people run some distributed service with Internet peers, that is still possible.
+
 >Since the location is coastal (e.g. wind, moisture, dense fog, animals, etc.), what are the allowances for HA - high availability - radios, routers/switches
 >- You can use simple STP for fail-over between radios
 >- 900Mhz can be a useful--if quite slow--backup
+>- Be wary of using any 60Ghz PtP option as fog can really hinder link performance
+
+We need the core areas to have redundancy against at least one radio failure, and have ways to monitor the overall link status of the network.
+
+We should get some [disaster.radio](https://disaster.radio) distributed at key locations, perhaps as a separate network where people can connect to a camp-global chat.
 
->Be wary of using any 60Ghz PtP option as fog can really hinder link performance
+Currently not using any 60 GHz equipment, but even if we do they'd be used (~ 250 m) at much smaller distances than specifications (1-2 km).
 
 >Is there power at each desired "node" location?
 >- If not, what is planned?
 >- POE beyond 100-150 feet is unreliable
 
+Not all locations have power. I believe the area with buildings and around the lake has power sources. At the end of March we will visit the site to identify power requirements. We also expect power build-out to complete before meshnet build-out.
+
 >Services
 >- Are any mesh services expecting to be reachable from the public internet?
 >- Waterproof/moisture-proof cases for Pi-like devices is crucial
 
+I don't expect services to be reachable automagically (as in our infrastructure will assign public IPs), but I fully expect people will manually peer stuff to their Internet peers and burn up our precious egress :cry:
+
 >LAN topology
 >- Will the network be "flat", or will there be vlan-segmented networks?
 >- Will these vlans need firewall policies between them?
 >- Will there be QoS in place for WAN egress traffic like VOIP/video-calling?
 
+Inclined to start very simple, and add things as needed. In April there will be a call to describe the basic network and how communities can deploy services on there (thinking to run stuff on Raspberry Pis or boxes they bring in). If there are specific requirements we can discuss whether to modify the network topology to make things happen.
+
 >Is the group planning on collecting data and visualizing these metrics?
 >- SNMP based monitoring platforms - LibreNMS, PRTG
 >- Input data into InfluxDB for data retrieval
 >- Dashboards like Grafana, Kabana, etc.
```

Comment:
I have some familiarity with Prometheus + Grafana on ARM devices. I think they have trouble when without a NTP server. We probably need one of those :)

I think InfluxDB and SNMP-based solutions are all good options. Someone who knows more about these can weigh in?

Reply:
SNMP is going to be readily available by most vendors of systems/networking equipment. A lot of the OIDs that they announce are readily standardized and documented. It's a old system, but used extensively and relied upon in the industry. 

The nice thing about databases like InfluxDB, is that you can have your SNMP monitoring tool populate it, or you can have a series of distributed scripts populating it for those devices/metrics that aren't SNMP-ready.

Why not build [your own SNMP server](https://developers.redhat.com/blog/2017/02/22/how-to-build-a-stratum-1-ntp-server-using-a-raspberry-pi/)? This has been a project I've been wanting to implement on my personal network for some time.

Reply:
I remember choosing Prom > Influx because I wanted something pull-based for my use case, but perhaps in this network it's better to use a push-based monitoring system? I think what you suggested (SNMP populating Influx) makes sense. Does it use a Grafana type of visualization tool?

I think it's super cool to have a local DIY NTP server. It's one of those "hidden infrastructure" that we just never think about. It'd be cool to be able to point at this thing and explain to Camp participants how it works and its role in our network!

Reply:
I logged the NTP one https://github.com/dweb-camp-2019/meshnet/issues/9

@raylas do you want to send a "GitHub suggestion" to answer the monitoring question?